plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/providers/vfs/OverwriteAwareFileSelector.java

```diff
@@ -41,16 +39,30 @@ public class OverwriteAwareFileSelector implements FileSelector {
   OverwriteStatus overwriteStatus;
   FileObject copyFrom;
   FileObject copyTo;
+
+  /**
+   * Separate VFS connection name variable is no longer needed.
+   * @deprecated
+   * The connection name is in the URI since full {@value org.pentaho.di.connections.vfs.provider.ConnectionFileProvider#SCHEME } paths are being used.
```

Comment:
similar comment to https://github.com/pentaho/pentaho-kettle/pull/9310/files#r1594093304